actionlist/app/reducers/action_items.js

```diff
@@ -8,15 +8,15 @@ export const defaultState = {
   drawerExpanded: false,
   optionSelected: '',
   dropdownColors: {
-    saved: Colors.White,
```

Comment:
What's the purpose of these changes? Having capitalized object keys seems pretty non-standard.